core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/JDBCExecutor.java

```diff
@@ -1,9 +1,93 @@
 package ca.jrvs.apps.jdbc;
 
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.List;
+
 public class JDBCExecutor {
 
   public static void main(String[] args) {
+    DatabaseConnectionManager dcm = new DatabaseConnectionManager("localhost",
+        "hplussport", "postgres", "password");
+    try{
+      Connection connection = dcm.getConnection();
+      CustomerDAO customerDAO = new CustomerDAO(connection);
+      /*Creating data example...
+      Customer customer = new Customer();
+      customer.setFirstName("Edward");
+      customer.setLastName("Wang");
+      customer.setEmail("EdwardWang@jrvs.ca");
+      customer.setPhone("(647) 416-6666");
+      customer.setAddress("1234 That Rd");
+      customer.setCity("Toronto");
+      customer.setState("ON");;
+      customer.setZipCode("M1G1Q1");
+
```

Comment:
could remove comments in code and put them in README

Reply:
I just uncomment those code sections instead of putting them in the README. Would that work?

Reply:
It will be good